toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -200,11 +234,24 @@ func constructVerityKernelCmdlineArgs(verityMetadata map[string]verityDeviceMeta
 			return nil, err
 		}
 
+		options := formattedCorruptionOption
+		if metadata.hashSignatureInjection != "" {
+			options += fmt.Sprintf(",root-hash-signature=%s", metadata.hashSignatureInjection)
+			hasSignatureInjection = true
+		}
+
 		newArgs = append(newArgs,
 			fmt.Sprintf("%s=%s", hashArg, metadata.rootHash),
 			fmt.Sprintf("%s=%s", dataArg, formattedDataPartition),
 			fmt.Sprintf("%s=%s", hashKey, formattedHashPartition),
-			fmt.Sprintf("%s=%s", optionsArg, formattedCorruptionOption),
+			fmt.Sprintf("%s=%s", optionsArg, options),
+		)
+	}
+
+	if hasSignatureInjection {
+		newArgs = append(newArgs,
+			"dm_verity.require_signatures=1",
```

Comment:
What if the user specifies two verity devices, one with a signature file and one without? Should we just let the user add this manually if they want it?

Reply:
Sure, removed, will add the instruction to the doc.